src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,158 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+    }
+
+    public class Node<T> {
```

Comment:
Inner Node should be private
```suggestion
    private class Node<T> {
```
All inner classes should be at the end of class

Reply:
Okay, I got you. Done.